tests/test.cpp

```diff
@@ -493,8 +493,21 @@ void ReflectionTest(uint8_t *flatbuf, size_t length) {
   TEST_NOTNULL(pos_table_ptr);
   TEST_EQ_STR(pos_table_ptr->name()->c_str(), "MyGame.Example.Vec3");
 
+  
+
   // Now use it to dynamically access a buffer.
   auto &root = *flatbuffers::GetAnyRoot(flatbuf);
+
+  {
+    // Verify the buffer first using reflection based verification
+    flatbuffers::Verifier v(flatbuf, length);
+    TEST_EQ(flatbuffers::Verify(v,
```

Comment:
This will no longer be required as we will create verifier internally inside reflection.cpp Verify function. But even there I have to create verifier like above because inorder to create it inline the VerifyObject function needs to take in verifier as const& but later I call function table->VerifyTableStart which expects a non const& to verifier and I don't want to use const cast there to make things work.